### DISCUSSION TITLE
How to set base url for upload files?

### DISCUSSION THREAD

**Discussion by vasilievruslan** [strapi/strapi#7436] [Legacy Topics]
I have difficulties to set base url for upload files with local provider. I receive /upload/hash.ext is it possible to do https://api.mydomain.com/upload/hash.ext and how? please help

**Comment by lauriejim** [comment]
Hello! By default, there is no option for that.
This is managed by the default upload provider - https://github.com/strapi/strapi/blob/master/packages/strapi-provider-upload-local/lib/index.js#L47
You can fork it and create a new one, you will be able to find how to do that into the documentation of the upload plugin.
This will update the stored `url` during the upload.
